pkg/syncer/metadatasyncer.go

```diff
@@ -304,6 +304,14 @@ func InitMetadataSyncer(ctx context.Context, clusterFlavor cnstypes.CnsClusterFl
 			}()
 		}
 		if isPodVMOnStretchSupervisorFSSEnabled {
+			log.Info("Loading CnsVolumeInfo Service to persist mapping for VolumeID to storage policy info")
```

Comment:
Do we need to start CnsVolumeInfo service for single cluster Supervisor as well when FSS is enabled?

Reply:
I think we need a check for single/multiple cluster supervisor at a common place, and need to address this for all of the CR updates & volume operations. Let's follow-up on this in the next PRs